examples/llava/llava.h

```diff
@@ -19,6 +19,10 @@
 
 struct clip_ctx;
 
+struct uhd_image_embed {
+    std::vector<std::vector<struct llava_image_embed *>> image_embeds;
+};
+
```

Comment:
Should stay C compatible, can maybe be declared similar to clip_ctx ?

Reply:
ok